### PR TITLE
APK Build on CI

### PR DESCRIPTION
**Pull request by coaguila**
# APK Build on CI
## Description
This PR introduces a new Github Actions Workflow to build an APK on each push on to `origin/main`. It closes issue #54.
## Changes
None
#### Added
- `.github/workflows/AndroidBuild.yml`
## Dependencies Added
Not applicable
## Testing
Not applicable

### CODE REVIEW

**Review by charliemangano — APPROVED** [review]
LGTM! thanks a lot

File: .github/workflows/AndroidBuild.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: AndroidBuild
+
+on:
+  pull_request:
+    branches: [ master ]
```
Comment (francelu) [inline .github/workflows/AndroidBuild.yml:5]: `main` not `master`

**Review by francelu — CHANGES_REQUESTED** [review]
LGTM.

File: .github/workflows/AndroidBuild.yml (hunk 1 above)
Comment (coaguila) [reply .github/workflows/AndroidBuild.yml:5]: you are correct, thank you for your input 👍🏽

**Review by francelu — APPROVED** [review]
LGTM.

**Pull request merged**